dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatCompletionServiceChatClient.cs

```diff
@@ -40,7 +40,7 @@ public ChatCompletionServiceChatClient(IChatCompletionService chatCompletionServ
         Verify.NotNull(chatMessages);
 
         var response = await this._chatCompletionService.GetChatMessageContentAsync(
-            new ChatHistory(chatMessages.Select(ChatCompletionServiceExtensions.ToChatMessageContent)),
+            new ChatHistory(chatMessages.Select(m => ChatCompletionServiceExtensions.ToChatMessageContent(m))),
```

Comment:
Is this change required?

Reply:
It is because the method accepts two parameters instead of one.